dunesim/EventGenerator/NuE/NuEScatterGen.fcl

```diff
@@ -29,7 +29,7 @@ standard_nuescattergen_solar:
    NuFlavBR:          [ 1., 0., 0., 0., 0., 0. ]
    FlatESpectrum:     false # generate flat energy spectrum for neutrino
 
-   EventRateFileName: "Flux/Solar/solar_nu-e_rates.root"
+   EventRateFileName: "/pnfs/dune/persistent/stash/Flux/Solar/v1/solar_nu-e_rates_KamLAND_bestfit.root"
```

Comment:
This should use the cvmfs stash path
`/cvmfs/dune.osgstorage.org/pnfs/fnal.gov/usr/dune/persistent/stash/Flux/Solar/v1/solar_nu-e_rates_KamLAND_bestfit.root`

Reply:
Path has been updated